cnv/cnv_release_notes/cnv-release-notes.adoc

```diff
@@ -27,10 +27,13 @@ include::modules/technology-preview.adoc[leveloffset=+2]
 // CNV-4134 virtctl upload
 
 // CNV-4080 VMI exposed in UI
+* You can view and manage virtual machine instances that are independent of virtual machines within the virtual machine lists.
 
 // CNV-4079 CD-ROM added to VM wizard
+* {CNVProductNameStart} enables you to configure a CD-ROM in the new *Virtual Hardware* page that is now available in the virtual machine wizard.  You can select the type of CD-ROM configuration from a drop-down list:  *Container*, *URL* or *Attach Disk*. You can also edit CD-ROM configurations in the *Virtual Machine Details* page.
```

Comment:
It reads like the Virtual Machine Details page is also part of the wizard.
Maybe we can change the first line to "...configure a CD-ROM in the virtual machine wizard." - unless someone specified that they want the Virtual Hardware part of the wizard called out.

Reply:
Simplifying the introductory sentence as noted by @aburdenthehand .